packages/@ama-sdk/schematics/README.md

```diff
@@ -64,27 +64,49 @@ npx -p @angular/cli ng add @ama-sdk/core
 
 ### How to use?
 
-The typescript generator provides 2 generators:
+The typescript generator provides 3 generators:
 
 - **shell**: To generate the "shell" of an SDK package
-- **core**: To (re)generate the SDK based on a specified Swagger spec
+- **core**: To (re)generate the SDK based on a specified OpenApi specification
 - **create**: To create a new SDK from scratch (i.e. chain **shell** and **core**)
 
-To generate the `shell` you can run:
+You can generate the `shell` in an existing monorepo with the command:
```

Comment:
the command to generate in an existing monorepo is 
`yarn ng g sdk typescript sdkName`
or with schematics
`yarn schematics @o3r/workspace:sdk sdkName`